lib/plugins/base.py

```diff
@@ -0,0 +1,66 @@
+from html.parser import HTMLParser
+
+from markdown import markdown
+
+
+# inspired by:
+# https://stackoverflow.com/questions/14694482/
+class HTMLFilter(HTMLParser):
+    text = ""
+    href = ""
+    li_count = 0
+    inside_blockquote = False
+    last_start_tag = ""
```

Comment:
Why not using `__init__` method here?
```suggestion
class HTMLFilter(HTMLParser):
    def __init__(self):
        super().__init__()
        self.text = ""
        self.href = ""
        self.li_count = 0
        self.inside_blockquote = False
        self.last_start_tag = ""
```

Reply:
yes, would be cleaner, was just lazy :-)